### PR TITLE
Add KIWI dataset

### PR DESCRIPTION
**Pull request by the-infiltrator**
image

All tests pass, have added metadata for the dataset as well to the best of my ability

### CODE REVIEW

File: data_summaries/KIWI.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        ],
+        "Task Categories": [
+            "Question Answering",
+            "Scientific Knowledge Retrieval",
+            "Question Understanding",
+            "Open Question Answering",
+            "Academic Tasks"
+        ],
+        "Text Sources": [
+            "User interactions",
```
Comment (shayne-longpre) [inline data_summaries/KIWI.json:25]: Will need to add this in here too: https://github.com/Data-Provenance-Initiative/Data-Provenance-Collection/blob/main/constants/domain_groups.json#L323

File: data_summaries/KIWI.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            "Generated text"
```
Comment (shayne-longpre) [inline data_summaries/KIWI.json:26]: Don't need this, as we have the "Model Generated" values

File: data_summaries/KIWI.json [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        "Human Annotation": "Yes",
+        "Derived from Datasets": [],
+        "Creators": [
+            "The University of Texas at Austin",
+            "University of Washington"
+        ],
+        "Licenses": [
+            {
+                "License": "CC BY-SA 4.0",
+                "License URL": "https://creativecommons.org/licenses/by-sa/4.0/legalcode"
```
Comment (shayne-longpre) [inline data_summaries/KIWI.json:45]: This URL instead, as we can see where its stipulated for the dataset: https://huggingface.co/datasets/fangyuan/kiwi#licensing-information

File: data_summaries/KIWI.json [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+            }
+        ],
```
Comment (shayne-longpre) [inline data_summaries/KIWI.json:47]: Add OpenAI and Llama 2 license to this list.

File: src/downloaders.py [hunk 5]
```diff
@@ -759,76 +870,93 @@ def download_gorilla(accepted_filter_ids):
...
-        dset = huggingface_download('MBZUAI/Bactrian-X', name=dset_name, split="train")
+        dset = huggingface_download("MBZUAI/Bactrian-X", name=dset_name, split="train")
         # annotate each example with source
         dset = annotate_source(dset, dset_name)
         dsets.extend(dset)
     return dsets
 
+
+def download_kiwi(accepted_filter_ids):
+    # dset = huggingface_download("internlm/Agent-FLAN")
```
Comment (shayne-longpre) [inline src/downloaders.py:960]: Delete

**Review by shayne-longpre — APPROVED** [review]
Great! Some minor fixes, and also take a look at the generated output and make sure it's well formed, then looks good to merge! :)

File: data_summaries/KIWI.json (hunk 4 above)
Comment (the-infiltrator) [reply data_summaries/KIWI.json:47]: Added openai need help for llama.

File: src/downloaders.py (hunk 5 above)
Comment (the-infiltrator) [reply src/downloaders.py:960]: Resolved.

File: data_summaries/KIWI.json (hunk 3 above)
Comment (the-infiltrator) [reply data_summaries/KIWI.json:45]: Resolved

**Comment by the-infiltrator** [comment]
image

**Pull request merged**